src/state/preferences/feed-tuners.tsx

```diff
@@ -0,0 +1,45 @@
+import {FeedTuner} from '#/lib/api/feed-manip'
+import {FeedDescriptor} from '../queries/post-feed'
+import {useLanguagePrefs} from './languages'
+
+export function useFeedTuners(feedDesc: FeedDescriptor) {
```

Comment:
Do you expect there'll be more user-specific stuff here pulled from context etc? If not I think I'd prefer this to just be `getFeedTuners(...)` and then the rest is filled in at the callsite. So that we don't get into a situation where we want this but it only works in a component.

Reply:
We will need to make the final call depending based on how the UX feels. If we want to automatically trigger re-fetches due to changes in the user preferences, then we need them in a hook. If we dont, we can just use a getter and then the user will refresh their feed manually.